### PR TITLE
Issue 88 non acgu characters

### PR DESCRIPTION
**Pull request by BurkovBA**

### CODE REVIEW

File: rnacentral/portal/models.py [hunk 1]
```diff
@@ -178,6 +185,23 @@ def count_symbols(self):
             'N': len(seq) - (count_A + count_C + count_G + count_U)
         }
 
+    def count_symbols(self):
```
Comment (blakesweeney) [inline rnacentral/portal/models.py]: Why not a one liner like:

```python
count = collections.Counter(self.get_sequence())
```

Comment (BurkovBA) [inline rnacentral/portal/models.py]: Wow, because I've never used it. Awesome, thanks!

Comment (blakesweeney) [inline rnacentral/portal/models.py]: You're welcome, the code looks fine to me now.

Comment (blakesweeney) [inline rnacentral/portal/models.py]: Actually just tested out Counter a bit as I thought it had a simpler api and it does:

```python
>>> import collections as coll
>>> coll.Counter('abbbbaa')
Counter({'b': 4, 'a': 3})
>>> dict(coll.Counter('abbbbaa'))
{'a': 3, 'b': 4}
>>> coll.Counter('abbbbaa')['a']
3
>>> coll.Counter('abbbbaa')['c']
0
```

no need for the call to `most_common` at least. Converting to a dict may or may not matter.

**Comment by AntonPetrov** [comment]
@blakesweeney here is an example sequence for testing: URS00000006AB

**Comment by blakesweeney** [comment]
Thanks, is there a search that quickly finds that in the site?

Also, I'm not sure the warning should be in grey. That seems to be used for additional information, not warnings. I'm open to suggestions on this, maybe orange?

**Comment by AntonPetrov** [comment]
No search like on the website, just in the database: `select * from rna where regexp_like(seq_short, '[bdefhijklmnopqrsvwxyz]', 'i');`

Yeah, maybe try different colours from http://getbootstrap.com/css/#helper-classes-colors

**Pull request merged**